students/kosarzewski_maciej/lesson_04_unit_testing/collatz/collatz_sequence.py

```diff
@@ -0,0 +1,17 @@
+def collatz(number):
+    if number <= 0:
+        raise ValueError
+    elif number % 2 == 0:
+        half_even = number / 2
+        print(half_even)
+        return half_even
+    elif number % 2 == 1:
```

Comment:
It's better to replace elif with else here. If you really want to keep elif than please remember to handle default case with else
e.g.
else:
    rasie ValueError